src/index.js

```diff
@@ -79,4 +79,4 @@ tasks.forEach((task) => {
 });
 saveData();
 document.querySelector('.add-btn').addEventListener('click', addToList);
-document.querySelector('.removeButton').addEventListener('click', removeList);
```

Comment:
- [ ] I couldn't find a way to edit a task from the interface of your app and it is a project requirement to have that

Reply:
- [ ] As suggested by the previous reviewer. Please make sure to implement the `edit` functionality as it's a project requirement also help your users to edit an already added `todo` instead of deleting and creating a new one each and every time they make a typo or something. See below for more details on the requirement:
<img width="562" alt="Screenshot 2023-05-11 at 23 11 46" src="https://github.com/LethaboMot/weppack2/assets/107192076/0831c3ef-eea3-4672-972d-9470bce1abe2">